docs/hooks.md

```diff
@@ -0,0 +1,34 @@
+# Snapcraft hook support
+
+Snapcraft supports snapd's [hooks][1] mechanism with two conventions:
```

Comment:
s/"with two conventions"/"using two possible methods"
Convention feels mandatory and makes the numbered list look like a list of steps.